models/openacademy_course.py

```diff
@@ -8,3 +8,14 @@ class Course(models.Model):
     description = fields.Text()
     responsible_id = fields.Many2one("res.users", ondelete="restrict")
     session_ids = fields.One2many("openacademy.session", string="Sessions", inverse_name="course_id")
+
+    _sql_constraints = [("description_and_title_different", "CHECK(name != description)", "The title and description of the course must be different."), ("unique_course_name", "UNIQUE(name)", "The name of the course must be unique.")]
+
+    def copy(self, cr, uid, id, default={}, context=None):
```

Comment:
This is the old API used until v8, from v9 the new format is used that includes those extra parameters in the self. 
Same for the other cases.

```suggestion
    def copy(self, default=None)
```

Reply:
Okay, got it.